Clash.hs

```diff
@@ -33,17 +33,17 @@ import Util (OverridingBool(..))
 genSystemVerilog
   :: String
   -> IO ()
-genSystemVerilog = doHDL (initBackend WORD_SIZE_IN_BITS HDLSYN True PreserveCase Nothing (AggressiveXOptBB False) :: SystemVerilogState)
+genSystemVerilog = doHDL (initBackend WORD_SIZE_IN_BITS HDLSYN True PreserveCase Nothing (AggressiveXOptBB False) (TernaryOpt True) :: SystemVerilogState)
```

Comment:
I think this has reached a breaking point. Could we refactor this into a record?

Reply:
Did someone say _refactor this_ :sparkles:

Reply:
:tada: